.github/workflows/deploy.yml

```diff
@@ -0,0 +1,58 @@
+name: Deploy
+
+on:
+  push:
+    branches:
+      - master
+  release:
+    types:
+      - published
+
+jobs:
+  build:
+    if: github.repository == 'RonnyPfannschmidt/iniconfig'
+    runs-on: ubuntu-20.04
+
+    steps:
+      - uses: actions/checkout@v2
+      - run: |
+            git fetch --prune --unshallow
+
+      - name: Cache
+        uses: actions/cache@v2
+        with:
+          path: ~/.cache/pip
+          key: deploy-${{ hashFiles('**/setup.py') }}
+          restore-keys: |
+            deploy-
+
+      - name: Set up Python
+        uses: actions/setup-python@v2
+        with:
+          python-version: 3.9
+
+      - name: Install dependencies
+        run: |
+          python -m pip install -U pip
+          python -m pip install -U setuptools twine wheel
```

Comment:
i just realized, we really should put setuptools_scm in here to avoid the easy_install
im hoping for a simple pep517 action in future

Reply:
Can do, do you mean this in `setup.cfg` isn't enough?

```cfg
setup_requires =
    setuptools_scm
```

Reply:
the problem is, if its in setup.cfg, setuptools will easy_install setuptools_scm instead of pip installing it
pre pyprojec.toml+preexisting wheels one would have to do a multi step install to get a working setuptools_scm on ci

its critical to avoid easy_install, critical enough to leave it out of setup_requires

Reply:
OK, like https://github.com/RonnyPfannschmidt/iniconfig/pull/20/commits/73f802bcbec58e8e4aafd753a057e1c956718ce6?